### DISCUSSION TITLE
Open the pdf at a particular page?

### DISCUSSION THREAD

**Discussion by zutamacka** [quasarframework/app-extension-qpdfviewer#119] [Q&A]
Hello! Wondering if it's possible to force the pdf to open at a particular page at this time? 

I've been reading the pdf in [Example 3](https://qpdfviewer.netlify.app/examples/pdfjs-example3) and tried feeding it something like src="../mypdfs/mytestpdf.pdf#page=2". It opens the pdf fine, but it ignores the page instruction (opens at page 1 or the last page I left off at, but not page 2).

**Comment by hawkeye64** [comment]
Seems reasonable. I will need to investigate this a bit more.

**Reply by zutamacka** [reply]
Thank you so much! :)

**Reply by hawkeye64** [reply]
My apologies as I have not had time to look into this. I am donating my previous laptop, so the dev environment is gone and I have not had time to set it up on the new laptop yet. As soon as I am able, I will look into this.

**Reply by zutamacka** [reply]
Hello, I am very grateful you are willing to tackle this at all :) Thanks again.
